src/rt/backtrace/dwarf.d

```diff
@@ -0,0 +1,464 @@
+/**
+ * This code handles backtrace generation using dwarf .debug_line section
+ * in ELF files for linux.
```

Comment:
In the dwarf module, the pattern of the output of `backtrace_symbol` function. In the elf module, the linux elf headers and usage of `/proc/self/exe`.
But most importantly, I didn't test on other systems.